tools/pwsh/clone_herbert_branch.ps1

```diff
@@ -14,13 +14,14 @@ param(
 . $PSScriptRoot/powershell_helpers.ps1
 
 $HERBERT_URL = "https://github.com/pace-neutrons/Herbert.git"
+$HERBERT_DIR = "Herbert"
 
 Write-Output "Building Herbert branch $branch..."
-if (Test-Path -Path "./Herbert") {
-  Invoke-In-Dir -directory "Herbert" -command "git fetch origin"
-  Invoke-In-Dir -directory "Herbert" -command "git checkout origin/$branch"
+if (Test-Path -Path "$HERBERT_DIR") {
```

Comment:
is the change from `-Path "./Herbert"` to `-Path "Herbert"` safe?

Reply:
Should be, it's still looking in the working directory